xml/System.Threading.Tasks/TaskCompletionSource.xml

```diff
@@ -192,7 +194,7 @@ The <xref:System.Threading.Tasks.Task> created by this instance and accessible t
         <remarks>To be added.</remarks>
         <exception cref="T:System.ArgumentNullException">
           <paramref name="exceptions" /> is <see langword="null" />.</exception>
-        <exception cref="T:System.ArgumentException">There are one or more <see langword="null" /> elements in <paramref name="exceptions" />.</exception>
+        <exception cref="T:System.ArgumentException">There is one or more <see langword="null" /> elements in <paramref name="exceptions" />.</exception>
```

Comment:
I think `are` may be correct, currently it reads "There is (x) elements", I think it is more correct to say "There are (x) elements". As for "There is one or more (x)" vs "There are one more (x)" I'm not sure but even then the latter sounds more correct to me and (x) then must be plural.

Reply:
Based on what I can tell from https://english.stackexchange.com/questions/13284/which-is-correct-one-or-more-is-or-one-or-more-are, it should be:

1) There is one or more...

-and-

2) One or more are...

Based on proximity of the noun to the verb. In the first example, "one" is closer to the verb. In the second example, "more" is closer to the verb.

Reply:
@mairaw Any thoughts?

Reply:
> https://english.stackexchange.com/questions/13284/which-is-correct-one-or-more-is-or-one-or-more-are
>  the verb should be singular, as long as the things in the list are individually singular

Based on this it sounds to me like "one or more" would be a list of one singular and one plural term. I haven't read into it much though.

Reply:
```suggestion
        <exception cref="T:System.ArgumentException">There are one or more <see langword="null" /> elements in <paramref name="exceptions" />.</exception>
```